docs/howto/krustlet-on-microk8s.md

```diff
@@ -9,6 +9,20 @@ MicroK8s and the Krustlet, on a single machine. `kubectl` is required but is ins
 as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity.
 You may use a standlone `kubectl` if you prefer.
 
+In order for the bootstrap authentication token to work, your kube-apiserver needs to have
+the `--enable-bootstrap-token-auth`
+(see [bootstrap-tokens](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/)).
+To verify you have it enabled, check the process args:
```

Comment:
can you add a newline character here so this starts on a new paragraph?

Reply:
can you also define "it"? i.e. "To verify you have the bootstrap authentication feature enabled"

This gives the user more context on what noun they should have enabled. Otherwise they may be confused whether you're asking for verification on the bootstrap authentication feature being enabled, the feature flag itself, or something else.

Reply:
good point